TradeItIosTicketSDK2/TradeItYahooBrokerSelectionViewController.swift

```diff
@@ -91,23 +91,34 @@ class TradeItYahooBrokerSelectionViewController: TradeItYahooViewController, UIT
     }
 
     private func getBroker(atIndexPath indexPath: IndexPath) -> TradeItBroker {
-        if !self.featuredBrokers.isEmpty && indexPath.section == 0 {
+        if self.isFeaturedBrokerSelected(indexPath: indexPath) {
             return self.featuredBrokers[indexPath.row]
         } else {
             return self.brokers[indexPath.row]
         }
     }
 
+    private func isFeaturedBrokerSelected(indexPath: IndexPath) -> Bool {
```

Comment:
Maybe rename to `isFeaturedCell(indexPath: IndexPath)` otherwise it could be confusing if someone selects a Featured Broker in the non-featured list.